data/tutorials/gs_00_up_and_running.md

```diff
@@ -338,7 +338,7 @@ When we change our program, we can type `dune build` again to make a new
 executable. To run the program, we can use:
 
 ```
-$ dune exec ./bin/main.exe
+$ dune exec ./_build/default/bin/main.exe
```

Comment:
If I run the guide's command step-by-step, it works for me:

```
~/Workspace/demo
➜ dune init project helloworld
Success: initialized project component named helloworld

~/Workspace/demo
➜ cd helloworld

Workspace/demo/helloworld
➜ dune build

Workspace/demo/helloworld
➜ dune exec ./bin/main.exe
Hello, World!
```

Can you paste your command history and the error you have?

Reply:
No worries! It was a bit surprising as `dune exec`'s goal is specifically to avoid having to write the path of the executables in the build dir. But bugs or errata in the doc can happen, so don't hesitate to open an issue if you hit another error 🙂